### DISCUSSION TITLE
Complex Conditions (nested if) should be supported

### DISCUSSION THREAD

**Discussion by marcelooblan2016** [vueform/builder#92] [Ideas]
Hi, We are working on a project which we need complex condition.
Currently form builder were unable to meet the condition we have: 
```
report_type = ['current owner search', 'vendee/lessee', 'preliminary']
  or
  ( report_type = 'document retrieval' and has_report = 'yes' )
  or 
  ( report_type = 'full' and any_deeds = 'yes')
and report_type = 'mortgage assignment chain'
```
![image](https://github.com/user-attachments/assets/2ddaa662-90c6-4732-b388-54b401a6aae0)
It would be great if you add another "And" button besides "Or" in that way a nested if can be achieved. Not sure how complicated that would be on the backend.

Temporarily, we have created multiple element to meet each conditions.
Hopefully, This feature will be added.

Let me know if you have other questions.

Best regards,
Marcelo

**Comment by dan-dazlab** [comment]
@adamberecz I would like to have a function on the conditions with acces by path to options values for the better control.

**Reply by adamberecz** [reply]
@dan-dazlab please open new discussion for a new topic. This is about nested conditions and yours is about accessing option values in conditions.

Also, if you can elaborate a bit more on what and how you want to use, what's it for, maybe giving a few short examples that would be helpful.
